src/main/java/io/pinecone/clients/Inference.java

```diff
@@ -1,20 +1,21 @@
 package io.pinecone.clients;
 
-import org.openapitools.control.client.ApiClient;
-import org.openapitools.control.client.ApiException;
-import org.openapitools.control.client.api.InferenceApi;
-import org.openapitools.control.client.model.EmbedRequest;
-import org.openapitools.control.client.model.EmbedRequestInputsInner;
-import org.openapitools.control.client.model.EmbedRequestParameters;
-import org.openapitools.control.client.model.EmbeddingsList;
+import io.pinecone.configs.PineconeConfig;
+import okhttp3.OkHttpClient;
+import org.openapitools.inference.client.ApiClient;
```

Comment:
example of updating import statement since inference is broken out of control module